packages/app/src/cli/models/organization.ts

```diff
@@ -44,6 +51,7 @@ export type OrganizationApp = MinimalOrganizationApp & {
   }
   configuration?: SpecsAppConfiguration
   flags: Flag[]
+  developerPlatformClient?: DeveloperPlatformClient
```

Comment:
I've checked it again, but there's an issue with the tests... Imagine this situation:
```
const APP = testOrganizationApp() // this should include a new developerPlatformClient instance
const developerPlatformClient = testDeveloperPlatformClient({
    async appFromId() => Promise.resolve(APP),
  })
```

The app returned by the developerPlatformClient would include a different developerPlatformClient instance. This would break many tests and the solution is not simple...